caffe2/CMakeLists.txt

```diff
@@ -770,7 +770,7 @@ if(USE_OPENMP AND OPENMP_FOUND)
   message(STATUS "pytorch is compiling with OpenMP. \n"
     "OpenMP CXX_FLAGS: ${OpenMP_CXX_FLAGS}. \n"
     "OpenMP libraries: ${OpenMP_CXX_LIBRARIES}.")
-  target_compile_options(torch_cpu INTERFACE ${OpenMP_CXX_FLAGS})
+  target_compile_options(torch_cpu PRIVATE ${OpenMP_CXX_FLAGS})
```

Comment:
Actually, I'm okay with it if you disable it at the user side. But, some internal libs may rely on this and I'm worried about that.

Reply:
Isn't torch_cpu one of the last libraries to be generated? I will try to generate a dependency graph and report back.